scripts/publish_prod.sh

```diff
@@ -44,11 +56,14 @@ else
     yarn standard-version --release-as $VERSION
 fi
 
-echo 'Publishing to npm'
-yarn test
+echo "Publishing to npm"
 yarn build
```

Comment:
`yarn build` has a step where it runs the tests so its unnecessary to have the explicit `yarn test` call